server/converters/cxgtool.py

```diff
@@ -409,6 +412,7 @@ def save_metadata(container, metadata):
     with tiledb.DenseArray(a_name, mode="w") as A:
         A.meta["cxg_version"] = CXG_VERSION
         A.meta["cxg_properties"] = json.dumps(metadata)
+        A.meta["colors"] = json.dumps(anndata_colors_to_cxg_colors(adata))
```

Comment:
lastly, this should only be stored in the CXG if the cxgtool user has _asked_ for it to be stored (just like title and about are optional).

Please add a CLI param to turn this feature on/off.